05/DB.test.js

```diff
@@ -128,6 +143,17 @@ describe('DB func update', () => {
 
 describe('DB fun truncate', () => {
 	it('func truncate should return clear array', async () => {
-		expect(await DBClass.truncate()).toBe(true);
+		const clearedArr = [ ];
+		const greatTests= {}
+
+		await DBClass.truncate();
+		const rows = await DBClass.getRows();
+		// myslalem, ze to bedzie super logika zeby sprawdzic czy tablica jest pusta
+		// expect(rows).toMatchObject([]);
+		expect(rows).toMatchObject(clearedArr);
+		
+		// ale to tez przechodzi :') wiec o co chodzi?
+		// expect(rows).toMatchObject(greatTests)
+		// expect(rows).toMatchObject({})
```

Comment:
Chodzi o to, że porównując obiekty porównujemy ich zawartość. Brak zawartości to właśnie co porównujesz ;P
Osobiście sprawdziłbym czy `expect(rows.length).toBe(0)` - najprościej jak można :)